index.js

```diff
@@ -0,0 +1,29 @@
+const { parse } = require('csv-parse')
+const fs = require('fs')
+const habitablePlanets = []

```

Comment:
lets move this into the readable stream functions scope. This is currently file level scope but its only used within the read stream function